profiles/coreos/targets/generic/make.defaults

```diff
@@ -1,8 +1,8 @@
 # Copyright (c) 2012 The Chromium OS Authors. All rights reserved.
 # Distributed under the terms of the GNU General Public License v2
 
-USE="cros-debug acpi usb symlink-usr usrmerge cryptsetup policykit"
```

Comment:
Is that now the default?

Reply:
No, `usrmerge` was replaced by `split-usr` upstream, which has an inverted meaning.  https://github.com/gentoo/gentoo/commit/f602af077d978ba8982ced38021ab97e90b7fcca  Our systemd ebuild was the last place using `usrmerge`, so we can just drop it.